components/fieldlabel/stories/fieldlabel.stories.js

```diff
@@ -104,6 +104,8 @@ RightAligned.storyName = "Right-aligned";
  * Field labels for required inputs can be marked with an asterisk at the end of the label. Optional inputs would then be understood to not have the asterisk. If using the asterisk icon, do not leave any space between the label text and the start of the `<svg>` element in the markup. Extra space should not be added in addition to the inline margin.
  *
  * The field label for a required field can display either the text “(required)”, or an asterisk.
+ *
+ * When marking a field as required, a non-breaking space (e.g. U+00A0 or &nbsp;) should be placed between the label text and the required marker to prevent the marker from appearing on its own line.
```

Comment:
```suggestion
 * When marking a field as required, a non-breaking space (e.g. `U+00A0` or `&nbsp;`) should be placed between the label text and the required marker to prevent the marker from appearing on its own line.
```
Thanks for adding this! One minor suggestion for formatting as code, otherwise the `&nbsp;` renders as HTML.